ts/sagas/startup/checkAcknowledgedFingerprintSaga.ts

```diff
@@ -0,0 +1,82 @@
+import { Effect } from "redux-saga";
+import { call, put, select, take } from "redux-saga/effects";
+import { getType } from "typesafe-actions";
+
+import { navigateToOnboardingFingerprintScreenAction } from "../../store/actions/navigation";
+import { fingerprintAcknowledge } from "../../store/actions/onboarding";
+
+import TouchID, { IsSupportedConfig } from "react-native-touch-id";
+import { isFingerprintAcknowledgedSelector } from "../../store/reducers/onboarding";
+import { GlobalState } from "../../store/reducers/types";
+
+export type BiometrySimpleType =
+  | "FINGERPRINT"
+  | "FACE_ID"
+  | "TOUCH_ID"
+  | "NOT_ENROLLED"
+  | "UNAVAILABLE";
+
+export function* checkSupportedFingerprintSaga(): IterableIterator<Effect> {
```

Comment:
I think we need to change the name of this method, it's not just a check, it has a navigation side effect - that is not totally obvious at the caller site - what about `onboardFingerprintIfAvailableSaga` ?

Reply:
also a short comment that describes the purpose of this function would be nice

Reply:
changed